test/testutil.c

```diff
@@ -73,6 +73,8 @@ int run_tests(const char *test_prog_name)
             }
         } else {
             for (j = 0; j < all_tests[i].num; j++) {
+                if (j == 625)
```

Comment:
Doh! Yes, there is a reason. It's debug code that crept in! It shouldn't be there.